src/slic3r/GUI/GLCanvas3D.cpp

```diff
@@ -2368,8 +2372,21 @@ void GLCanvas3D::reload_scene(bool refresh_immediately, bool force_full_scene_re
             }
         }
         for (int temp_idx = vol_idx; temp_idx < m_volumes.volumes.size() && !update_object_list; temp_idx++) {
-            if (!m_volumes.volumes[temp_idx]->is_wipe_tower)
+            // Volumes in m_volumes might not exist anymore, so we cannot
+            // directly check if they are is_wipe_towers, for which we do
+            // not want to update the object list.  Instead, we do a kind of
+            // slow thing of seeing if they were in the deleted list, and if
+            // so, if they were a wipe tower.
+            bool was_deleted_wipe_tower = false;
+            for (int del_idx = 0; del_idx < deleted_volumes.size(); delidx++) {
```

Comment:
How did this ever work?  I cannot understand what happened.  This *did* compile once, at least, I think, though.  I'll have a fixup commit for this in a moment.

Reply:
Fixed.  I have no idea how I ever got this to compile to begin with.  Sigh.